fe/fe-core/src/main/java/com/starrocks/server/CatalogMgr.java

```diff
@@ -147,9 +147,10 @@ public void replayCreateCatalog(Catalog catalog) throws DdlException {
         if (Strings.isNullOrEmpty(type)) {
             throw new DdlException("Missing properties 'type'");
         }
-        if (!CreateCatalogStmt.SUPPORTED_CATALOG.contains(type)) {
```

Comment:
I think you need to remove the SUPPORTED_CATALOG and use SUPPORT_CONNECTOR_TYPE to instead of it.  So you don't have to change the code with the version.